docs/api/inputs/button.md

```diff
@@ -8,8 +8,23 @@
 
 ```python
 @app.cell
-def __():
-    mo.ui.button(label="Click me")
+def _():
+    import marimo as mo
+    return (mo,)
```

Comment:
Our docs plugin automatically adds an `import marimo as mo` block, so you should actually delete this.